dist/render.js

```diff
@@ -0,0 +1,44 @@
+class Renderer {
+    constructor() {
+        const source = $('#player-template').html();
+        const statsSource = $('#stats-template').html();
+        const myTeamsSource = $('#team-template').html();
+        this._template = Handlebars.compile(source);
+        this._statsTemplate = Handlebars.compile(statsSource);
+        this._myTeamsTemplate = Handlebars.compile(myTeamsSource);
+    }
+
+    showElement = element => element.css("visibility", "visible")
+    hideElement = element => element.css("visibility", "hidden")
+
+    _handleBarHelper = (elementToAppendTo, Template, data) => {
+        elementToAppendTo.empty();
```

Comment:
Remove this line and on line 17 you can chain the methods:

`elementToAppendTo.empty().append(newHTML)`